src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -0,0 +1,72 @@
+/** @jsx h */
+import { h, Fragment } from 'preact';
+import { useState } from 'preact/hooks';
+import Icon from '../../Icon';
+import Instructions from '../../Instructions';
+import Donut from '../../Donut';
+import ProductListLink from '../../ProductListLink';
+import styles from './styles.scss';
+
+import { useServerData } from '../../../lib/providers';
```

Comment:
```suggestion
import { useServerData } from '../../../lib/providers';
```
Since this no longer will be used, we can remove it.